src/StackAdmin/Azs.Subscriptions.Admin/custom/GeneratedCmdlets.txt

```diff
@@ -0,0 +1,1230 @@
+<#
+.Synopsis
```

Comment:
why this text file is needed?

Reply:
not needed. this file contains the default generated cmdlets for all the customized ones.